Common/Sources/Extensions/String.swift

```diff
@@ -84,17 +84,14 @@ public extension String {
     /// Get FX Rate with four decimal
     /// - Returns: return string with four decimal
     func getFxRateWithFourDecimal() -> String {
-        guard !self.isEmpty else {
-            return ""
-        }
-        let numbers = self.split(separator: ".")
-        if numbers.count == 2 {
-            let decimals = numbers[1]
-            if decimals.count > 4 {
-                return String(self.dropLast(decimals.count - 4))
-            }
-        }
-        return self
+        guard !self.isEmpty
+        else { return "0" }
```

Comment:
Is there any specific reason, this return statement is changed?

Reply:
It should still be "" and not 0

Reply:
Ok, will update